app/forms/submissions/trial/step03_content_form.rb

```diff
@@ -1,8 +1,9 @@
 module Submissions
   module Trial
     class Step03ContentForm < PlantTrialForm
-      # NOTE: needed because completely empty step breaks submission
-      property :_i_am_fake
+      property :lines_or_varieties, writeable: false
```

Comment:
Does not persist this value in associated active record in the DB. However, you we probably don't need it here as you do not "associate" that form with any particular AR class, right? I mean, you use "reform" in a slightly different way, correct?